test/SecTester.Scan.Tests/ScanTests.cs

```diff
@@ -1,53 +1,228 @@
+using Microsoft.Extensions.Logging;
+using NSubstitute.ClearExtensions;
+using SecTester.Scan.Tests.Fixtures;
+
 namespace SecTester.Scan.Tests;
 
-public class ScanTests
+public class ScanTests : ScanFixture
```

Comment:
@derevnjuk I do not see any need to use `IClassFixture` since these tests do not require sharing any expensive context and moreover require isolation from each other. I will rework the code to get rid of inheritance

Reply:
@ostridm so, then it is not clear why you create a shared `ScanFixture` if you have to isolated context. This is a broken design.